wagtail_streamfield_migration_toolkit/migrate_operation.py

```diff
@@ -0,0 +1,166 @@
+import json
+from django.db.models import JSONField, F, Q
+from django.db.models.functions import Cast
+from django.db.migrations import RunPython
+from wagtail.blocks import StreamValue
+
+from wagtail_streamfield_migration_toolkit import utils
+
+
+class MigrateStreamData(RunPython):
+    """Subclass of RunPython for streamfield data migration operations"""
+
+    def __init__(
+        self,
+        app_name,
+        model_name,
+        field_name,
+        operations_and_block_paths,
+        revisions_from=None,
+        chunk_size=1024,
+        **kwargs
+    ):
+        """MigrateStreamData constructor
+
+        Args:
+            app_name (str): Name of the app.
+            model_name (str): Name of the model.
+            field_name (str): Name of the streamfield.
+            operations_and_block_paths (:obj:`list` of :obj:`tuple` of (:obj:`operation`, :obj:`str`)):
+                List of operations and corresponding block paths to apply.
+            revisions_from (:obj:`datetime`, optional): Only revisions created from this date
```

Comment:
Nitpick: this still isn't 100% clear that live and latest revisions will always be updated even if they're created before this date.